kitsune/groups/views.py

```diff
@@ -47,6 +48,7 @@ def profile(request, group_slug, member_form=None, leader_form=None):
             "members": members,
             "user_can_edit": user_can_edit,
             "user_can_manage_leaders": user_can_manage_leaders,
+            "user_can_message_group": user_can_message_group,
```

Comment:
No need for the var. It's not used anywhere else
`"user_can_message_group": request.user.profile.is_staff`